package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "medic-conf",
-  "version": "2.0.4",
+  "version": "2.0.5",
```

Comment:
FYI of new steps for releasing `medic-conf`. https://github.com/medic/medic-conf#releasing. You no longer need to make this change as part of the PR. Once this PR is approved, use the `npm version` command to publish and bump.